enterprise-suite/values.yaml

```diff
@@ -106,12 +104,15 @@ prometheusMemoryRequest: 250Mi
 # Persistent Volumes
 #
 # Set to true to use persistent volumes, otherwise emptyDir volumes are used.
-usePersistentVolumes: false
-# If true, the chart manages persistent volumes by creating PVCs. Set to false to not manage PVCs at all.
-# Warning: If you "unmanage" PVCs that helm is managing by setting this to false, helm will delete those PVCs.
-managePersistentVolumes: true
-# Default StorageClass to use for persistent volumes
-defaultStorageClass: standard
+usePersistentVolumes: true
+# Default StorageClass to use for persistent volumes.  If unspecified, use the cluster's DefaultStorageClass.
+#defaultStorageClass: standard
+# Application-specific storage classes. If unspecified (and usePersistentVolumes is true), the application-specific
+# PVCs will use the defaultStorageClass setting.
+#prometheusStorageClass:
+#esAlertmanagerStorageClass:
+#esGrafanaStorageClass:
```

Comment:
This is all WIP?

Reply:
Nope.  Everything commented out now.  That will use defaults.  User needs to tweak to use their own storage class(es).